extension/chrome/settings/modules/help.ts

```diff
@@ -73,7 +73,7 @@ View.run(
       await Ui.delay(50); // give spinner time to load
       try {
         const res = await this.externalService?.helpFeedback(emailVal, `${textVal}\n\n\nFlowCrypt ${Catch.browser().name} ${VERSION}`);
-        if (res?.sent) {
+        if (res?.status_code === 200) {
```

Comment:
there is no need for exact check if `status_code` is equal to `200`, we already have check for `response.ok` at https://github.com/FlowCrypt/flowcrypt-browser/blob/8140affdd2cba6983e75ee47b5ffd7a3bdcb1f05/extension/js/common/api/shared/api.ts#L178, which checks if response was successful (https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) and throws `AjaxErr` otherwise.

so here we should just check `if (res)`

Reply:
I get the point. This PR is not really good so I'll be closing this. Simple PR would be at https://github.com/FlowCrypt/flowcrypt-browser/pull/5540.